### PR TITLE
feat: support call kclvm service by go rpc server

### PR DESCRIPTION
**Pull request by NeverRaR**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):

internal/kclvm_py/scripts/
kclvm/capi/src/service/service.rs
#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other
chorm: build kclvm capi lib and cpython shared lib for kclvm

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: internal/kclvm_py/scripts/update-kclvm.sh [hunk 1]
```diff
@@ -117,16 +117,83 @@ if [ -e $topdir/kclvm/target/release/libkclvm.dll ]; then
...
+fi
+# Windows dll
+if [ -e $topdir/kclvm/target/release/libkclvm_capi.dll ]; then
+    touch $kclvm_install_dir/lib/libkclvm_capi.dll
+    rm $kclvm_install_dir/lib/libkclvm_capi.dll
+    cp $topdir/kclvm/target/release/libkclvm_capi.dll $kclvm_install_dir/lib/
+    cp $topdir/kclvm/target/release/libkclvm_capi.dll $kclvm_install_dir/lib/libkclvm_capi.dll
+fi
+
+#build capi go rpc server
```
Comment (Peefy) [inline internal/kclvm_py/scripts/update-kclvm.sh:191]: move go rpc server to `kclvm-go` repo.

File: internal/kclvm_py/scripts/update-kclvm.sh [hunk 2]
```diff
@@ -117,16 +117,83 @@ if [ -e $topdir/kclvm/target/release/libkclvm.dll ]; then
...
+cp $topdir/gorpc/gorpc $kclvm_install_dir/bin/
+rm $topdir/gorpc/gorpc
+
+
 # Copy LLVM runtime and header
+cd $topdir/kclvm/runtime
 cp src/_kclvm.bc $kclvm_install_dir/include/_kclvm.bc
 cp src/_kclvm.h  $kclvm_install_dir/include/_kclvm.h
 
+cd $topdir/kclvm/capi
```
Comment (Peefy) [inline internal/kclvm_py/scripts/update-kclvm.sh:165]: duplicated capi lib build sctips.

File: gorpc/main.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package main
```
Comment (Peefy) [inline gorpc/main.go:1]: move go-rpc server to `kclvm-go` repo.

**Review by Peefy — APPROVED** [review]
LGTM

**Review by chai2010 — APPROVED** [review]
LGTM

**Pull request merged**